### PR TITLE
Add module for extended worker

### PR DESCRIPTION
**Pull request by debelyoo**
## Purpose
Use siddhi worker with custom extensions

### CODE REVIEW

File: dockerfiles/worker-extended/init.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
```
Comment (a-pagano) [inline dockerfiles/worker-extended/init.sh:1]: Why is this file needed?

Comment (debelyoo) [reply dockerfiles/worker-extended/init.sh:1]: It is used in the Docker file

**Review by a-pagano — APPROVED** [review]
